pkg/database/migrations.go

```diff
@@ -2181,7 +2181,11 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 					`ALTER TABLE realm_stats
 						ALTER COLUMN user_reports_issued SET NOT NULL,
 						ALTER COLUMN user_reports_claimed SET NOT NULL,
-						ALTER COLUMN user_report_tokens_claimed SET NOT NULL`)
+						ALTER COLUMN user_report_tokens_claimed SET NOT NULL`,
+					`ALTER TABLE user_reports
```

Comment:
This needs to be a different migration since this migration has already been run

Reply:
fixed.